### ISSUE TITLE
Refocus opened diff view / don't open the same diff view multiple times

### ISSUE THREAD

**Issue by outdooracorn** [uwolfer/gerrit-intellij-plugin#433]
When I double click on a file in the list of changes for a patch it opens a new diff view even if that diff view is already opened in another tab. It would be good if double clicking on a file that is already opened in a tab focused that tab rather than duplicating it.

**Comment by uwolfer** [comment]
I think this is the same behavior as the IntelliJ diff viewer used for commis and change list.@outdooracorn  Can you confirm?

**Comment by outdooracorn** [comment]
If I understand you correctly, it's not the same behaviour as the IntelliJ diff viewer used for commits and change lists. Only a single diff view tab is used for all files in a commit or change view. For example, if I double click on a file in a change list a diff view tab opens. If I double click that file again nothing happens. If I double click another file in the change list it opens in the same diff view tab that is already open.

**Comment by uwolfer** [comment]
Ah, you are right. It depends on the diff viewer you are using in IntelliJ. This plugin uses the "old" one, which is still available also for change lists. But the new one is "inline", which does not have this issue. Would you prefer to have the "inline" one in this plugin as well?

**Comment by outdooracorn** [comment]
Yeah, I think the plugin should use the new "inline" diff viewer so it's consistent with the diff viewer used for commits.
